template-parts/content-blocks/block-accordion.php

```diff
@@ -13,15 +13,13 @@
 $row_index       = get_row_index();
 
 // Start a <container> with possible block options.
-_s_display_block_options(
-	array(
-		'container' => 'section', // Any HTML5 container: section, div, etc...
-		'class'     => 'content-block grid-container accordion-block', // Container class.
-	)
-);
+_s_display_block_options( array(
+	'container' => 'section', // Any HTML5 container: section, div, etc...
```

Comment:
@coreymcollins we might want to discuss this further. I think this would sort of justfiy having some function + acf field to determine the output. I could argue for div or even an aside or article depending on the content.

Reply:
The only one, based on intended use, that would for sure be a div or aside would be the cta block.

Reply:
I figured this would be one we could all chat about, so I added an agenda item for today's scrum. I'm sure we'll all come to a decision on what would be best for us + users.